mousestyles/visualization/distribution_plot.py

```diff
@@ -0,0 +1,80 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from mousestypes.est_power_param import (fit_powerLaw, fit_exponential,
+                                         getdistance, powerlaw_pdf, exp_pdf)
+
+
+def plot_powerlaw(estimation):
+    """
+    Return the histogram of all estimators of power law
+    to check the distribution.
+
+    Parameters
+    ----------
+    estimation: dataframe
+        dataframe of strain, mouse, day and the estimator
+
+    Returns
+    -------
+    plot : histogram
+        The histogram of all estimators of power law
+    """
+    plt.hist(list(estimation.ix[estimation["strain"] == 0, 3]))
+    plt.hist(list(estimation.ix[estimation["strain"] == 1, 3]))
+    plt.hist(list(estimation.ix[estimation["strain"] == 2, 3]))
+    plt.title("Histogram: Power Law parameters distribution by strain")
```

Comment:
Based on the discussion with Jarrod, I have changed all my functions name. One function is imported from yours so could you change you "fit_powerLaw" to "fit_powerlaw" to avoid future conflicts?